windows.sh

```diff
@@ -23,6 +23,7 @@ printf "\033[0m\n\n"
 
 # Check that add-apt-repository is installed for non-standard Vagrant boxes
 if [ ! -f /usr/bin/add-apt-repository ]; then
+  sudo apt-get update
   echo "Adding add-apt-repository..."
   sudo apt-get -y install software-properties-common
 fi
```

Comment:
I had thought about that, because they both do indeed run as you suspect on first `vagrant up` and adds to your initial provisioning time.

What I couldn't figure out was how to do both in one shot avoiding two `update`'s.  I needed the first one so I could first install `software-properties-common` (this problem is what started the linked discoure thread).  After adding the ansible repository you have to run `update` again to then install ansible itself.

I will push a change adding `-y` to the command to match the below one